benchmark/fft_batch/config.cc

```diff
@@ -0,0 +1,87 @@
+#include "benchmark/fft_batch/config.h"
+
+#include "tachyon/base/console/iostream.h"
+#include "tachyon/base/flag/flag_parser.h"
+
+namespace tachyon {
+namespace base {
+
+template <>
+class FlagValueTraits<Config::PrimeField> {
+ public:
+  static bool ParseValue(std::string_view input, Config::PrimeField* value,
+                         std::string* reason) {
+    if (input == "baby_bear") {
+      *value = Config::PrimeField::kBabyBear;
+    } else {
+      *reason = absl::Substitute("Unknown prime field: $0", input);
+      return false;
+    }
+    return true;
+  }
+};
+
+template <>
+class FlagValueTraits<Config::Vendor> {
+ public:
+  static bool ParseValue(std::string_view input, Config::Vendor* value,
+                         std::string* reason) {
+    if (input == "plonky3") {
+      *value = Config::Vendor::kPlonky3;
+    } else {
+      *reason = absl::Substitute("Unknown vendor: $0", input);
+      return false;
+    }
+    return true;
+  }
+};
+
+}  // namespace base
+
+// static
+std::string Config::VendorToString(Config::Vendor vendor) {
+  switch (vendor) {
+    case Config::Vendor::kPlonky3:
+      return "plonky3";
+  }
+  NOTREACHED();
+  return "";
+}
+
+bool Config::Parse(int argc, char** argv) {
+  base::FlagParser parser;
+
+  parser.AddFlag<base::Flag<size_t>>(&repeating_num_)
+      .set_short_name("-n")
+      .set_help("Specify the number of repetition 'n'. By default, 10.");
+  parser.AddFlag<base::Flag<bool>>(&check_results_)
+      .set_long_name("--check_results")
+      .set_help("Whether checks results generated by each fft_batch runner.");
+  parser.AddFlag<base::Flag<uint32_t>>(&degree_)
+      .set_short_name("-d")
+      .set_long_name("--degree")
+      .set_help("Specify the domain size. By default, 18.");
+  parser.AddFlag<base::Flag<size_t>>(&batch_size_)
+      .set_short_name("-b")
+      .set_long_name("--batch_size")
+      .set_help("Specify the batch size. By default, 100.");
+  parser.AddFlag<base::Flag<PrimeField>>(&prime_field_)
+      .set_short_name("-p")
+      .set_long_name("--prime_field")
+      .set_help(
+          "A prime field to be benchmarked with. (supported prime fields: "
+          "baby_bear");
+  parser.AddFlag<base::Flag<std::vector<Vendor>>>(&vendors_)
+      .set_long_name("--vendor")
```

Comment:
How about setting a default value since it only has one!

```suggestion
      .set_long_name("--vendor")
      .set_default_value(Vendors::kPlonky3)
```